core/computed/entity-classification.js

```diff
@@ -33,7 +36,7 @@ class EntityClassification {
       company: rootDomain,
       category: '',
```

Comment:
We can fill out the `category` and `homepage` fields for unrecognized chrome extension too now?

Reply:
Is that a likely possibility that we have Chrome extensions that are not in devToolsLogs as `Runtime.executionContextCreated`?

Reply:
I believe so. In timespan mode the execution context could already exist when the run starts. Then `Runtime.executionContextCreated` won't be in the DT log